Competition/src/main/include/Constants.h

```diff
@@ -241,7 +241,7 @@ namespace FeederConstants{
     constexpr static double DEFAULT_FEEDER_SPEED_REVERSE = -1.0;
 
     constexpr static int CACHE_SIZE = 20;
-    constexpr static double JAM_CURRENT = 30;
+    constexpr static double JAM_CURRENT = 20;
```

Comment:
When we run this is auto, does it stop the intake when it jumps off for the first time?